christfellowshipchurch/web-app-v2
# Added new video to external web page

### About
Added What to Expect video to External page

### Test Instructions
Go to cf.church, scroll down and you should see the video pop up in a modal once you click on the image of Pastors Todd and Julie

### Screenshots
![Screen Shot 2024-01-22 at 10 49 27 AM (1)](https://github.com/christfellowshipchurch/web-app-v2/assets/46769629/48d86e1b-0b0e-428d-aeb2-d32fe8f0fa53)
![Screen Shot 2024-01-22 at 10 49 37 AM](https://github.com/christfellowshipchurch/web-app-v2/assets/46769629/357f8667-6e94-474f-99e9-fb9b53195aaf)

### Closes Tickets
[CFDP-2871](https://christfellowshipchurch.atlassian.net/jira/software/projects/CFDP/boards/2?selectedIssue=CFDP-2871)

### Related Tickets
N/A


[CFDP-2871]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2871?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
